webapp/src/components/expanded_view/expanded_view.scss

```
@@ -0,0 +1,20 @@
+@keyframes callsKeyframesPulsing {
```

Stylelint (17.14.1)

[error] Expected keyframe name "callsKeyframesPulsing" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)